notebooks/GoPro2CropKMZ.ipynb

```diff
@@ -0,0 +1,897 @@
+{
```

Comment:
<p>This comment "Once finished, enter the verification code provided in your browser:" confused me a bit because for a while i was looking for where to enter the code.</p><p> </p><p>Therefore, i think this comment should be direct, for eg. "Once finished, enter the verification code provided in your browser <strong>here</strong>: ---"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/street2sat/pull/51/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>